packages/contracts-bedrock/deploy-config/mainnet.ts

```diff
@@ -3,11 +3,15 @@ import { DeployConfig } from '../src/deploy-config'
 // NOTE: The 'mainnet' network is currently being used for bedrock migration rehearsals.
 // The system configured below is not yet live on mainnet, and many of the addresses used are
 // unsafe for a production system.
+
+const foundationMultisig = 'f39fd6e51aad88f6f4ce6ab8827279cfffb92266'
+const feeRecipient = '0x90F79bf6EB2c4f870365E785982E1f101E93b906'
+
 const config: DeployConfig = {
-  finalSystemOwner: '0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A',
-  controller: '0x78339d822c23d943e4a2d4c3dd5408f66e6d662d',
-  portalGuardian: '0x78339d822c23d943e4a2d4c3dd5408f66e6d662d',
```

Comment:
Did you mean to alter the values here?

Reply:
Yes, `0x783` was a local address I used. `0x9ba` is the real foundation multisig which prevented us from unpausing the portal before. These addresses should all be the same, even when we go to mainnet, but for rehearsal they will just be the hh test addresses.